internal/backend/mqttpubsub/backend.go

```diff
@@ -101,6 +105,11 @@ func NewBackend(c BackendConfig) (*Backend, error) {
 	opts.SetOnConnectHandler(b.onConnected)
 	opts.SetConnectionLostHandler(b.onConnectionLost)
 
+	if b.config.LastWillPayload != "" {
+		log.WithField("topic", b.config.LastWillTopic).Info("backend: setting last will message to mqtt broker")
+		opts.SetWill(b.config.LastWillTopic, b.config.LastWillPayload, b.config.LastWillQoS, b.config.LastWillRetain)
```

Comment:
Please note that you still have to render the topic! E.g. see the `b.ConfigTemplate, err = template.New("config")....` line above this change.